docs/portal/registry-users/viewing-users.md

```diff
@@ -0,0 +1,16 @@
+# Viewing Registry Users
+
+The following procedure describes how to view information about registry users. Buttons at the top right of the displayed form allow you to view logs associated with the registry user, delete the registry user, and update the registry user.
```

Comment:
Please change "to view logs associated with the registry user" to "to change password for the registry user"

Reply:
I changed your suggestion subtly to the following:" ...to change the password of the registry user..."